### ISSUE TITLE
workarea v 3.4.26 compatibility 

### ISSUE THREAD

**Issue by roseliux** [workarea-commerce/workarea-orderbot#15]
I need to use this plugin in my current project that has workarea 3.4.26
I update workarea to 3.5 but seems to not be a straight forward solution because of dependencies is there any reason why this is not compatible with version 3.4 ? 
any suggestion is welcome, thanks.

**Comment by jyucis** [comment]
Hi,

The Orderbot integration relies heavily on the v3.5 self configuration system.  If you wanted to make this v3.4 compatible you could modify:

https://github.com/workarea-commerce/workarea-orderbot/blob/master/config/initializers/configurations.rb

You would need to check to see if `Workarea` responded to `define_fields`. If it does not you could just manually create the configuration the way you would in a v3.4 application. 

Feel free to create a PR.

**Comment by roseliux** [comment]
open this PR https://github.com/workarea-commerce/workarea-orderbot/pull/16 
and the Workarea don't respond to define fields.

**Comment by bencrouse** [comment]
@roseliux see @jyucis 's message above. This would be a `if` statement you could wrap around the configuration field definition.

**Comment by roseliux** [comment]
ey @bencrouse it should be something like:
```
if Workarea::Configuration.respond_to?(:define_fields)
  code
else
  Workarea.config.orderbot  = { api_user_name: 'string', api_password: 'string }
 end
```

the else clause is what is not clear to me, it looks correct?

**Comment by bencrouse** [comment]
Closing since this was added in #16 and released in v1.0.1.
